library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +71,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    Set<DayViewDecorator> dayViewDecorators;
```

Comment:
I'm not certain what you mean by avoid duplication. Could you give a short example?